lib/lambda-cfn.js

```diff
@@ -385,7 +385,11 @@ function lambda(options) {
     var validRuntimes = ['nodejs','nodejs4.3'];
     if (validRuntimes.indexOf(options.runtime) === -1 ) {
       throw new Error('Invalid AWS Lambda node.js runtime ' + options.runtime);
+    } else {
+      fn.Properties.Runtime = options.runtime;
     }
+  } else {
+    fn.Properties.Runtime = 'nodejs4.3';
```

Comment:
It may be worth moving the validRuntimes array + string for 'nodejs4.3' up as global variables with an explicit name like AWS_Default_Version for reuse/ ease of changing in the future